exasol/nb_connector/slc/script_language_container.py

```diff
@@ -327,6 +335,31 @@ def activation_key(self) -> str:
                 "Secure Configuration Storage does not contains an activation key."
             ) from ex
 
+    def generate_activation_key(self, add_to_secret_store: bool) -> str:
+        """
+        Generates the language activation string for the uploaded script-language-container.
+        Can be used in `ALTER SESSION` or `ALTER_SYSTEM` SQL commands to activate
+        the language of the uploaded script-language-container.
+        This method can be used as fallback if the deploy-function failed for some reason,
+        but the script-language-container was uploaded to the BucketFS.
```

Comment:
```suggestion
        Furthermore, this method can be used as fallback if the deploy-function failed for some reason,
        but the script-language-container was uploaded to the BucketFS. Then it will register the Language activation for this SLC in the Secure Configuration Store.
```